### DISCUSSION TITLE
How to select what to show in the standalone viewer?

### DISCUSSION THREAD

**Discussion by lypwig** [bernhard-42/vscode-ocp-cad-viewer#117] [Q&A] [answered]
I'm having some issues related to #112 and it seems that the standalone viewer works well as a workaround.

But I only see the OCP logo (which is nice but not very helpful), how to select my actual parts from the viewer?

**Comment by bernhard-42** [comment]
Not sure I understand you fully, but you need to add `from ocp_vscode import *` and use `show(obj1, ...)` in your code. 
It takes port 3939 as default. If you started the standalone with another port, use `set_port(my_port)` in your code

**Comment by lypwig** [comment]
Thank you. The standalone viewer is executed with the cli and doesn't take a file parameter.

If I have many Python files on the same folder from witch I run `ocp_vscode`, how the viewer can know what I want to see?

**Comment by bernhard-42** [comment, marked as answer]
The viewer listens on port 3939 for requests.
The `show` command tessellates CAD objects and sends the mesh to port 3939 of the viewer via websockets.
So viewer and code is completely decoupled. You can start the viewer where so ever and work on a project in another folder.
The only shared knowledge of the viewer and the `show` command is the websocket port 3939
